petstore/frontend/react/src/comments/Comments.js

```diff
@@ -0,0 +1,183 @@
+import React, {Component} from 'react';
+import {string} from 'prop-types';
+import AddComment from './AddComment';
+import Thread from './Thread';
+import config from '../config';
+
+class Comments extends Component {
```

Comment:
This is the only component that was difficult to switch to using hooks. That's because it defines a large number of methods that use props and state.